src/code/UpdatePSResource.cs

```diff
@@ -299,7 +299,13 @@ protected override void EndProcessing()
 
             if (installedPackages.Count is 0)
             {
-                WriteWarning($"No installed packages were found with name '{string.Join(",", namesToProcess)}' in scope '{Scope}'. First install package using 'Install-PSResource'.");
+                var message = $"No installed packages were found with name '{string.Join(",", namesToProcess)}' in scope '{Scope}'. First install package using 'Install-PSResource'.";

```

Comment:
Add a space after the comma, to improve readability

```suggestion
                var message = $"No installed packages were found with name '{string.Join(", ", namesToProcess)}' in scope '{Scope}'. First install package using 'Install-PSResource'.";

```